parlai/core/metrics.py

```diff
@@ -34,6 +34,100 @@
 }
 ALL_METRICS = DEFAULT_METRICS | ROUGE_METRICS | BLEU_METRICS | DISTINCT_METRICS
 
+MetricDisplayData = namedtuple('MetricDisplayData', ('title', 'description'))
```

Comment:
how about a data class instead

Reply:
Hm, what's the advantage you see? I was using a namedtuple because this isn't really data that should be mutable.

If it's the pretty syntax you're looking for, though, I just found out there's a nicer syntax for namedtuples:
```
class MetricDisplayData(NamedTuple):
    title: str
    description: str
```

Reply:
We used NamedTuples before and it gave me headaches and now i stay away with them. That syntax is nicer and fine with me.

Reply:
I'm curious what headaches you encountered! I haven't used them a ton (just for small things like this here and there) so maybe there're headaches incoming I'm ignorant of.